composer.json

```diff
@@ -91,12 +91,13 @@
         "drupal/field_defaults": "^2.0",
         "drupal/field_group": "^3.0",
         "drupal/field_permissions": "^1.0@RC",
+        "drupal/flexible_permissions": "^1.1",
         "drupal/focal_point": "^2.0",
         "drupal/geocoder": "^3.2 || ^4.0",
         "drupal/geofield": "^1.3",
         "drupal/google_tag": "^1.4",
-        "drupal/group": "^1.4",
-        "drupal/groupmedia": "^2.0@alpha",
+        "drupal/group": "2.2.2",
```

Comment:
Noticed that v2.2.2 just became unsupported. Can we update to v2.3.0?
https://pgov-1103-portlandor.pantheonsite.io/admin/reports/updates/update

Reply:
Ah, just saw the Group issue 3482418. Guess will update when the fix is merged.

Reply:
Yeah, we can update once that issue is fully resolved.